rita_common/src/dashboard/wallet.rs

```diff
@@ -54,8 +58,11 @@ fn withdraw_handler(address: Address, amount: Option<Uint256>) -> HttpResponse {
         };
 
     let tx_cost = gas_price * tx_gas;
-    if amount.clone() + tx_cost.clone() >= balance {
-        amount = balance - tx_cost;
+    if amount.clone() + tx_cost.clone() >= balance.clone().unwrap_or_else(|| 0u64.into()) {
```

Comment:
this if statement is a little overcomplicated, your unwrapping and then matching on the balance, the other way around would be cleaner.